### PR TITLE
Ad data on resolving events

### PR DESCRIPTION
**Pull request by clementFrancon**
### Description

In order to troubleshoot ad performances, we need to have some information on the current ad node being resolved 

### Type
- [ ] Breaking change
- [x] Enhancement
- [ ] Fix
- [ ] Documentation
- [ ] Tooling

### CODE REVIEW

File: src/parser/vast_parser.js [hunk 1]
```diff
@@ -108,13 +108,14 @@ export class VASTParser extends EventEmitter {
...
-   * @param {String} previousUrl - url of the previous VAST
+   * @param {Number} wrapperDepth - How many times the current url has been wrapped.
+   * @param {String} previousUrl - Url of the previous VAST.
+   * @param {Object} ad - Previously parsed ad node (Wrapper) related to this fetching.
    * @emits  VASTParser#VAST-resolving
    * @emits  VASTParser#VAST-resolved
    * @return {Promise}
    */
-  fetchVAST(url, wrapperDepth = 0, previousUrl = null) {
+  fetchVAST(url, wrapperDepth = 0, previousUrl = null, ad = null) {
```
Comment (ZacharieTFR) [inline src/parser/vast_parser.js:118]: What do you think about renaming `ad` to `wrapperAd` to make it more explicit (especially on the markdown documentation)
```suggestion
  fetchVAST(url, wrapperDepth = 0, previousUrl = null, wrapperAd = null) {
```

File: src/parser/vast_parser.js [hunk 2]
```diff
@@ -129,6 +130,7 @@ export class VASTParser extends EventEmitter {
         wrapperDepth,
         maxWrapperDepth: this.maxWrapperDepth,
         timeout: this.fetchingOptions.timeout,
+        ad,
```
Comment (clementFrancon) [inline src/parser/vast_parser.js:133]: not sure if we should call it ad or parentAd or something WDYT?

Comment (clementFrancon) [reply src/parser/vast_parser.js:133]: Wow 1 sec between our comments, ok I'll change it

**Review by ZacharieTFR — APPROVED** [review]
LGTM ! 🚀

**Pull request merged**